integration-tests/testkit/seed.ts

```diff
@@ -59,6 +72,16 @@ export function initSeed() {
 
           return {
             organization,
+            async setFeatureFlag(name: string, enabled: boolean) {
```

Comment:
I could do it over GraphQL API but we don't need to set features flags over a GraphQL request or any other form of remote calls. Once we have a need for a more complex admin panel, we could implement it, but until then, the feature flag might be no longer available.